chromium/USAGE.md

```diff
@@ -17,4 +17,27 @@ Please note that not all algorithm combinations are expected to work. Most notab
 
 Please create a [discussion item](https://github.com/open-quantum-safe/boringssl/discussions/landing) if you feel some algorithm combination that does not work should do.
 
-
+## Issues and Workarounds
+
+### Ubuntu/Deb
+
+`ERROR:nsNSSCertificateDB.cpp(95)] PK11_ImportCert failed with error -8168`
+
+There are a few things to try should you recieve this error whilst importing your proxy CA certificate:
+
+1. Manually install CA cert into nssdb.
+```sh
+# Change permissions to avoid error "read only database"
+# chmod -R 766 /home/username/.pki/
```

Comment:
Could you change `/home/username` to `$HOME`?

Reply:
needs to be run as sudo so not really.

Reply:
My bad. Everything looks good to me!